src/main/java/com/spotinst/sdkjava/model/bl/elastigroup/aws/RequiredVCpuMemoryGpu.java

```diff
@@ -0,0 +1,113 @@
+package com.spotinst.sdkjava.model.bl.elastigroup.aws;
+
+import com.fasterxml.jackson.annotation.JsonFilter;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.spotinst.sdkjava.client.rest.IPartialUpdateEntity;
+
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * @author: Anurag Sharma
+ * @since: 17/08/2023
+ */
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonFilter("PartialUpdateEntityFilter")
+public class RequiredVCpuMemoryGpu implements IPartialUpdateEntity {
```

Comment:
implements IPartialUpdateEntity not required for bl class

Reply:
Fixed.